modules/nerd-fonts/nerd-fonts.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -oue pipefail
+
+NERD_FONTS_DIR=/usr/share/fonts/nerd-fonts
+COMPRESS_TYPES=("tar.xz" "zip")
+
+mkcd() {
+
+    if [ -d "$1" ]; then
+        
+        rm -rf "$1"
+
+    fi
+    
+    mkdir -p "$1"
+    cd "$1" || exit 0
+
+}
+
+download() {
+
+    local downloaded
+    local file
+
+    mkcd "$1"
+
+    echo "${COMPRESS_TYPES[@]}" | while IFS=$'\n' read -r -d ' ' type; do
+
+        downloaded=$(ls | wc -l)
+        file="$1.$type"
+
+        if [[ $downloaded -eq 0 ]]; then
+
+            echo "--- Downloading $file ---"
```

Comment:
The ”---”’s signify the start of a block, the end being ”------” as per build.sh. This is just convention, but if you’re going to use a block inside a module, I think it should be a lever deeper with ”----” and use the same style of closing without any text inside the closing block.

This is mostly preference (of having the module output look the same), but also kind of my fault in not including more documentation around the conventions.

Reply:
I confess that I didn't understand this comment, it's probably my weak English hahaha. See what it looks like now with the new code.

Reply:
No problem, looks like you understood enough to do changes as needed.

What I meant is that output should look something like this, with the `----` block being a bit different from the `---` block used by build.sh and it would contain the output for the specific action and end with `--------` (2 times 4 dashes).
```
---- Downloading Google Fonts ----
<curl command output>
Downloaded font <fontname>
--------
```